src/page/VariantStore.ts

```diff
@@ -86,6 +86,13 @@ export class VariantStore {
     @observable public variant: string = '';
     @observable public selectedTranscript: string = '';
 
+    readonly genomeBuild: MobxPromise<string> = remoteData({
+        await: () => [this.mainStore.genomeBuild],
+        invoke: async () => {
+            return Promise.resolve(this.mainStore.genomeBuild.result);
+        },
+    });
```

Comment:
Should we just do this?

```suggestion
    readonly genomeBuild: MobxPromise<string> = this.mainStore.genomeBuild;
```

if not maybe this?

```suggestion
    readonly genomeBuild: MobxPromise<string> = remoteData({
        await: () => [this.mainStore.genomeBuild],
        invoke: () => Promise.resolve(this.mainStore.genomeBuild.result),
    });
```

Reply:
@onursumer 👍